Source/WebKit/Scripts/webkit/tests/TestWithEnabledByAndConjunctionMessageReceiver.cpp

```diff
@@ -42,13 +42,15 @@ void TestWithEnabledByAndConjunction::didReceiveMessage(IPC::Connection& connect
 {
     auto& sharedPreferences = sharedPreferencesForWebProcess();
     UNUSED_VARIABLE(sharedPreferences);
+    bool runtimeEnablementCheckFailed = false;
+    UNUSED_VARIABLE(runtimeEnablementCheckFailed);
     if (!(sharedPreferences.someFeature && sharedPreferences.otherFeature)) {
 #if ENABLE(IPC_TESTING_API)
         if (connection.ignoreInvalidMessageForTesting())
             return;
 #endif // ENABLE(IPC_TESTING_API)
         ASSERT_NOT_REACHED_WITH_MESSAGE("Message %s received by a disabled message receiver TestWithEnabledByAndConjunction", IPC::description(decoder.messageName()).characters());
-        return;
```

Comment:
It doesn't seem right that this return statement is going away.

Reply:
Yes, I caught this too; fixing